examples/bzlmod/BUILD.bazel

```diff
@@ -11,6 +11,8 @@ load("@python_3_9//:defs.bzl", py_test_with_transition = "py_test")
 load("@python_versions//3.10:defs.bzl", compile_pip_requirements_3_10 = "compile_pip_requirements")
 load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
 
+exports_files([".coveragerc"])
```

Comment:
I was expecting that this is not necessary, but it keeps failing without this

Reply:
It is not needed only if the default visibility is public within the BUILD.bazel file.